pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -38,6 +39,7 @@
 , passthru ? {}
 , pkgconfigDepends ? [], libraryPkgconfigDepends ? [], executablePkgconfigDepends ? [], testPkgconfigDepends ? []
 , testDepends ? [], testHaskellDepends ? [], testSystemDepends ? []
+, benchDepends ? []
```

Comment:
As mentioned elsewhere, this option should be called "benchmarkDepends", IMHO, for consistency.